scripts/build-images.sh

```diff
@@ -38,23 +40,23 @@ else
 fi
 
 echo "Starting image build"
-export REGISTRY=docker.io/
-export REPO=multicloudlab
+export REGISTRY=quay.io/cluster-api-provider-ibmcloud
 
-echo "Logging into registry ${REGISTRY///}"
-docker login -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}"
-
-echo "Building controller docker image"
+echo "Building controller and clusterctl docker images."
 cd ${base_dir}
-make container IMG=${REGISTRY}${REPO}/cluster-api-provider-ibmcloud-controller:${TAG}
+VERSION=$TAG make images
 cd -
 
-echo "Pushing image with tag '${TAG}'."
-docker push ${REGISTRY}${REPO}/cluster-api-provider-ibmcloud-controller:${TAG}
+echo "Logging into registry ${REGISTRY%%/*}"
+docker login -u "${REGISTRY_USERNAME}" -p "${REGISTRY_PASSWORD}"
 
-if [ "$LATEST" == "latest" ]; then
+echo "Pushing images with tag '${TAG}'."
+VERSION=$TAG make push-images
```

Comment:
looks good, just curious what's this line for?

Reply:
this script will build image and push it to the `quay.io` repo. So, this push-images will push the built image to the repo.

I guess the script name is not good. I will chnage it.